src/components/animated-image/animated-image.ts

```diff
@@ -7,6 +7,9 @@ import styles from './animated-image.styles';
 import type { CSSResultGroup } from 'lit';
 
 /**
+ * @title AnimatedImage
```

Comment:
@claviska 
I am not sure if such a title makes sense for You ...

![grafik](https://user-images.githubusercontent.com/4742603/196720148-f79237b6-267c-489b-8a0a-b826b9fa9b92.png)


Before the components name seems to be more usefull
![grafik](https://user-images.githubusercontent.com/4742603/196720440-dd75c834-0790-4161-baa9-aff2af7ad04b.png)

Reply:
The names prefixed with `Sl` are the class names, but they're also used in JSX for the React wrappers. Ideally, we'd continue to use the class names and not change anything.

Reply:
So it makes sense to remove `title` as `customTag` in `custom-elements-manifest.config.js` what was implement 3 days ago, Or would you keep it?